src/renderer/viz/expressions/binary.js

```diff
@@ -455,11 +455,6 @@ function genBinaryOp (name, allowedSignature, jsFn, glsl) {
             b = implicitCast(b);
 
             const signature = getSignatureLoose(a, b);
-            if (signature !== undefined) {
```

Comment:
Why is this removed?

Reply:
Because it was a psuedo `checkLooseType` check, see https://github.com/CartoDB/carto-vl/issues/878